documentation/markup.rst

```diff
@@ -678,8 +676,12 @@ Roles
 As said before, Sphinx uses interpreted text roles to insert semantic markup in
 documents.
 
-Names of local variables, such as function/method arguments, are an exception,
-they should be marked simply with ``*var*``.
+There are a couple of cases where roles can be omitted in favor of
+simpler markup:
```

Comment:
> In addition, there are a few places in the CPython documentation where simpler markup is used:

To me this sounds like "a simpler markup is used in some part of the CPython docs", whereas in theory these exceptions are/should be applied to all the CPython docs.  It also sounds like a neutral statement, whereas I wanted to tell the reader that they should do this too (the following two sentences are more explicit about this).  I agree that we should point out that this is a CPython-specific convention.

> avoid [...] contradicting the following statement implying these are roles (as they are)

Did you mean "aren't roles"?  `*...*` is not a role afaik, wheareas `` `...` `` technically is, even though from the context it can  be inferred that this is talking about `` :role:`...` `` roles.

Maybe something like `In the CPython documentation, simpler markup is conventionally used in a few common cases:` is better?

Reply:
> To me this sounds like "a simpler markup is used in some part of the CPython docs", whereas in theory these exceptions are/should be applied to all the CPython docs. It also sounds like a neutral statement, whereas I wanted to tell the reader that they should do this too (the following two sentences are more explicit about this). 

Good points. Here's a revision taking both of them and your suggestion into account:

```suggestion
In the CPython documentation, there are a few common cases
where simpler markup should be used:
```

> Did you mean "aren't roles"? `*...*` is not a role afaik, wheareas `` `...` `` technically is, even though from the context it can be inferred that this is talking about `` :role:`...` `` roles.

They are indeed roles; I didn't initially think they were either until I checked [the relevant reST docs](https://docutils.sourceforge.io/docs/ref/rst/roles.html#emphasis) as part of writing the above, and both `:emphasis:` and `:literal:` are indeed roles, with `*..*` and ``` ``...`` ``` as equivalent forms (just like `` `...` `` is for the default `title-reference` role).